lib/worker.py

```diff
@@ -114,7 +114,8 @@ def reproduce_task_groups(task_groups):
     found_keys = list(set(found_keys))
     if len(found_keys) < 1:
         raise ValueError('[reproduce] Cannot find any suite for given tests')
-    elif len(found_keys) > 1:
+    elif len(found_keys) == 2 and found_keys[1] != found_keys[0] + '_fragile' \
+            or len(found_keys) > 2:
```

Comment:
I would rather trim '_fragile' at end of keys inside the loop above. So this check will 'ignore' 'foo' and 'foo_fragile' difference and everything else should work as expected.

Reply:
Good idea
Thank you!